### PR TITLE
Prototype of drpc-benchmark.

### PR DESCRIPTION
**Pull request by zhengrenjie**
Run `sh test/script/benchmark/benchmark.sh` to get benchmark info.

### CODE REVIEW

File: test/script/benchmark.bash [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```
Comment (jovany-wang) [inline test/script/benchmark.bash]: rename to .sh

File: test/script/benchmark.bash [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
```
Comment (jovany-wang) [inline test/script/benchmark.bash]: Add a brief here to elaborate what this script is.

File: test/script/wrk.lua [hunk 3]
```diff
@@ -0,0 +1,20 @@
+done = function(summary, latency, requests)
```
Comment (jovany-wang) [inline test/script/wrk.lua]: Put the file and `benchmark.sh` to scripts/benchmark dir

File: test/src/main/resources/pb/benchmark.proto [hunk 4]
```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
```
Comment (jovany-wang) [inline test/src/main/resources/pb/benchmark.proto]: rename the dir `test/src/main/resources/proto/benchmark.proto`

File: test/src/main/java/com/distkv/drpc/test/common/Constants.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
+package com.distkv.drpc.test.common;
+
+public class Constants {
+
+  /**
+   * benchmark content.
+   */
+  public static final String TEST_CONTENT =
```
Comment (jovany-wang) [inline test/src/main/java/com/distkv/drpc/test/common/Constants.java]: I think we should define some contents with different sizes, like 5bytes, 100 bytes, 10m etc:

static final byte[]  TEST_CONTENT_5_BYTES = new byte[5];

static final byte[]  TEST_CONTENT_100_BYTES = new byte[100];

...

And then refine the proto to:
```proto
Message Request {
repeated byte xxx;
}
```

**Review by jovany-wang — COMMENTED** [review]
`.DS_Store` shouldn't be checked in.

**Comment by jovany-wang** [comment]
Some conflicts were introduced.

**Comment by zhengrenjie** [comment]
I have fixed.
Please check it again.
@jovany-wang @MrKingHH @senyer @kairbon

**Comment by zhengrenjie** [comment]
> `.DS_Store` shouldn't be checked in.

**Review by jovany-wang — CHANGES_REQUESTED** [review]
will share you more info offline.

**Comment by zhengrenjie** [comment]
Please check it again.
@jovany-wang

**Review by jovany-wang — APPROVED** [review]

**Pull request merged**